packages/js/product-editor/src/hooks/use-product-manager/use-product-manager.ts

```diff
@@ -45,6 +46,11 @@ function errorHandler( error: WPError, productStatus: ProductStatus ) {
 
 export function useProductManager< T = Product >( postType: string ) {
 	const [ id ] = useEntityProp< number >( 'postType', postType, 'id' );
+	const [ name, , prevName ] = useEntityProp< string >(
```

Comment:
Nitpicky, but I don't see a reason to abbreviate `previous to `prev` here.

Reply:
The pattern I'm using here (`prev[variable-name]`) is being used in multiple places. We didn't use the `previous` before, but I'm okay with renaming the variables that use the `prev` in the package. I would tackle that in a follow-up PR.